ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -401,6 +401,21 @@ public struct FluentNotification: View, TokenizedControlView {
     // When true, the notification will fit the size of its contents.
     // When false, the notification will be fixed based on the size of the screen.
     private let isFlexibleWidthToast: Bool
+
+    // MARK: Constants
+
+    /// The value for the horizontal padding between the elements within a notification and its frame
+    private static let horizontalPadding: CGFloat = GlobalTokens.spacing(.medium)
+
+    /// The value for the vertical padding between the elements within a multi-line notification and its frame
+    private static let verticalPadding: CGFloat = GlobalTokens.spacing(.small)
+
+    /// The value for the horizontal spacing between the elements within a notification
+    private static let horizontalSpacing: CGFloat = GlobalTokens.spacing(.medium)
```

Comment:
We had a specific ask for the horizontal spacing to be customizable, so we may want to keep this one.

Reply:
Looks like we need `minimumHeight` too :(